fundingmanager_test.go

```diff
@@ -953,15 +1068,45 @@ func TestFundingManagerRestartBehavior(t *testing.T) {
 		t.Fatalf("expected to not find channel state, but got: %v", state)
 	}
 
+	// Exchange the fundingLocked messages.
```

Comment:
An identical code block is copy-pasta'd verbatim above in `TestFundingManagerNormalWorkflow`. The logic should be factor out into something along the lines of `assertFundingFlowResolution`.